t/00-load.t

```diff
@@ -4,12 +4,13 @@ use strict;
 use warnings;
 use Test::More;
 
-plan tests => 3;
+plan tests => 4;
 
 BEGIN {
+    use_ok( 'Data::Chronicle' ) || print "Bail out!\n";
```

Comment:
I'd skip the "Bail out!" lines - we want that final diag() to run, since it'll give us Perl version info even if the module(s) didn't load. if you really want to bail out then might as well use `Test::More::BAIL_OUT()`, rather than relying on STDOUT parsing?